app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java

```diff
@@ -173,12 +173,24 @@ public void onStopTrackingTouch(SeekBar seekBar) {
         mMuseActions = new View[]{toggleButton, horizonButton};
 
         mConcentrationBar = view.findViewById(R.id.concentrationBar);
-
-        mColorMap = ColorMap.Greenish();
+        mConcentrationBar.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                changeColorMap();
+            }
+        });
 
         return view;
     }
 
+    synchronized void changeColorMap() {
```

Comment:
s/synchronized/private

This only gets called from the onClickListener so doesn't need to be synchronized.